test-data/pretty_format_yaml/multi-doc-not-pretty-formatted.yaml

```
@@ -0,0 +1,9 @@
+---
+root:
+    test:
+     -  1
```

I would make this doc very ugly, so the result will be more visible
```suggestion
root: {test: [1]}
```

---

I can make it more ugly, but your suggestion is already pretty-formatted according to the tool.

---

That's surprising ... but yeah. Let's ignore this comment.
I'm marking this as resolved